src/main/resources/assets/minecolonies/lang/en_US.lang

```diff
@@ -1,35 +1,32 @@
-com.minecolonies.error.supplyChestAlreadyPlaced=You have already placed a supply ship!
-
+com.minecolonies.error.supplyChestAlreadyPlaced=You have already placed a supply ship\!
```

Comment:
Do we need these characters escaped? Or should I add a line to the fix-lang tool to remove all \ characters?